% Options for packages loaded elsewhere
\PassOptionsToPackage{unicode}{hyperref}
\PassOptionsToPackage{hyphens}{url}
%
\documentclass[
]{book}
\title{Data Standards - TKI Indigenous Genomics}
\author{Sam Buckberry \& Jimmy Breen}
\date{2022-03-02}

\usepackage{amsmath,amssymb}
\usepackage{lmodern}
\usepackage{iftex}
\ifPDFTeX
  \usepackage[T1]{fontenc}
  \usepackage[utf8]{inputenc}
  \usepackage{textcomp} % provide euro and other symbols
\else % if luatex or xetex
  \usepackage{unicode-math}
  \defaultfontfeatures{Scale=MatchLowercase}
  \defaultfontfeatures[\rmfamily]{Ligatures=TeX,Scale=1}
\fi
% Use upquote if available, for straight quotes in verbatim environments
\IfFileExists{upquote.sty}{\usepackage{upquote}}{}
\IfFileExists{microtype.sty}{% use microtype if available
  \usepackage[]{microtype}
  \UseMicrotypeSet[protrusion]{basicmath} % disable protrusion for tt fonts
}{}
\makeatletter
\@ifundefined{KOMAClassName}{% if non-KOMA class
  \IfFileExists{parskip.sty}{%
    \usepackage{parskip}
  }{% else
    \setlength{\parindent}{0pt}
    \setlength{\parskip}{6pt plus 2pt minus 1pt}}
}{% if KOMA class
  \KOMAoptions{parskip=half}}
\makeatother
\usepackage{xcolor}
\IfFileExists{xurl.sty}{\usepackage{xurl}}{} % add URL line breaks if available
\IfFileExists{bookmark.sty}{\usepackage{bookmark}}{\usepackage{hyperref}}
\hypersetup{
  pdftitle={Data Standards - TKI Indigenous Genomics},
  pdfauthor={Sam Buckberry \& Jimmy Breen},
  hidelinks,
  pdfcreator={LaTeX via pandoc}}
\urlstyle{same} % disable monospaced font for URLs
\usepackage{longtable,booktabs,array}
\usepackage{calc} % for calculating minipage widths
% Correct order of tables after \paragraph or \subparagraph
\usepackage{etoolbox}
\makeatletter
\patchcmd\longtable{\par}{\if@noskipsec\mbox{}\fi\par}{}{}
\makeatother
% Allow footnotes in longtable head/foot
\IfFileExists{footnotehyper.sty}{\usepackage{footnotehyper}}{\usepackage{footnote}}
\makesavenoteenv{longtable}
\usepackage{graphicx}
\makeatletter
\def\maxwidth{\ifdim\Gin@nat@width>\linewidth\linewidth\else\Gin@nat@width\fi}
\def\maxheight{\ifdim\Gin@nat@height>\textheight\textheight\else\Gin@nat@height\fi}
\makeatother
% Scale images if necessary, so that they will not overflow the page
% margins by default, and it is still possible to overwrite the defaults
% using explicit options in \includegraphics[width, height, ...]{}
\setkeys{Gin}{width=\maxwidth,height=\maxheight,keepaspectratio}
% Set default figure placement to htbp
\makeatletter
\def\fps@figure{htbp}
\makeatother
\setlength{\emergencystretch}{3em} % prevent overfull lines
\providecommand{\tightlist}{%
  \setlength{\itemsep}{0pt}\setlength{\parskip}{0pt}}
\setcounter{secnumdepth}{5}
\usepackage{booktabs}
\usepackage{amsthm}
\makeatletter
\def\thm@space@setup{%
  \thm@preskip=8pt plus 2pt minus 4pt
  \thm@postskip=\thm@preskip
}
\makeatother
\ifLuaTeX
  \usepackage{selnolig}  % disable illegal ligatures
\fi
\usepackage[]{natbib}
\bibliographystyle{apalike}

\begin{document}
\maketitle

{
\setcounter{tocdepth}{1}
\tableofcontents
}
\hypertarget{introduction}{%
\chapter{Introduction}\label{introduction}}

Here we outline TKI Indigenous Genomics groups standards for data storage, access and sharing. Where possible, these standards will align with the The Global Alliance for Genomics and Health \href{https://www.ga4gh.org/}{GA4GH} standards.

\hypertarget{background}{%
\chapter{Background}\label{background}}

\hypertarget{data-descriptors-definitions}{%
\chapter{Data descriptors \& definitions}\label{data-descriptors-definitions}}

\textbf{File type:} This is the file format to be typically denoted by the file extension.\\

\textbf{Description:} A brief description of the file format.\\

\textbf{Compression:} The algorithm and/or method used to compress the file.\\

\textbf{Permissions:} The groups and users that have access to the data.\\

\textbf{Access frequency:} How often this data type may require access. These descriptions are based on a subset of the \href{https://cloud.google.com/storage/docs/storage-classes\#classes}{Google Cloud storage class descriptors}.

\begin{itemize}
\tightlist
\item
  \textbf{Standard:} Standard Storage is for data that is frequently accessed and/or stored for only brief periods of time. Standard data files should be able to be regenerated from Nearline or Archive data files without too much time or effort.
\item
  \textbf{Nearline:} Nearline Storage is ideal for data you plan to read or modify on average once per month or less. Nearline Storage is appropriate for data backup and short-term archiving.
\item
  \textbf{Archive:} highly durable storage for data archiving, backup, and disaster recovery. Only used for infrequently accessed data.
\end{itemize}

\textbf{Community access:} Are their community custodians for these data? If so, who are they?\\

\textbf{Identifiable:} Could this data be used to identify an individual (YES/NO).\\

\textbf{Location:} Location(s) where the data should be stored.\\

\textbf{Notes:} Any additional information that would assist users of the data.\\

\textbf{Sample:} A sample of the file data with descriptions of each field. Useful if this file type is not a widely used and documented standard.

\hypertarget{whole-genome-sequencing}{%
\chapter{Whole genome sequencing}\label{whole-genome-sequencing}}

\hypertarget{cram}{%
\section{CRAM}\label{cram}}

\textbf{File type:} CRAM\\
\textbf{Description:} CRAM supports a wide
range of lossless and lossy sequence data preservation strategies enabling users to choose which data should be preserved. \href{https://www.ga4gh.org/cram/}{CRAM is the genomics compression standard for GA4GH}\\
\textbf{Compression:} NA\\
\textbf{Permissions:}\\
\textbf{Community access:}\\
\textbf{Location:}
\textbf{Notes:} When used with a reference genome, this exact reference genome file should be recorded.\\
\href{https://samtools.github.io/hts-specs/CRAMv3.pdf}{CRAM specification}

\hypertarget{vcf}{%
\section{VCF}\label{vcf}}

\textbf{File type:} VCF\\
\textbf{Description:}\\
\textbf{Compression:}\\
\textbf{Permissions:}\\
\textbf{Access frequency:}\\
\textbf{Community access:}\\
\textbf{Identifiable:}\\
\textbf{Location:}\\
\textbf{Notes:}

\hypertarget{dna-methylation}{%
\chapter{DNA methylation}\label{dna-methylation}}

\begin{center}\rule{0.5\linewidth}{0.5pt}\end{center}

\hypertarget{raw-data}{%
\section{Raw data}\label{raw-data}}

\hypertarget{fastq}{%
\subsection{FASTQ}\label{fastq}}

\textbf{File type:} FASTQ\\
\textbf{Description:}\\
\textbf{Compression:}\\
\textbf{Permissions:}\\
\textbf{Access frequency:}\\
\textbf{Community access:}\\
\textbf{Identifiable:}\\
\textbf{Location:}\\
\textbf{Notes:}

\begin{center}\rule{0.5\linewidth}{0.5pt}\end{center}

\hypertarget{mapped-data}{%
\section{Mapped data}\label{mapped-data}}

\hypertarget{cram-1}{%
\subsection{CRAM}\label{cram-1}}

\textbf{File type:} CRAM\\
\textbf{Description:} Genomic alignments (typically). CRAM supports a wide
range of lossless and lossy sequence data preservation strategies enabling users to choose which data should be preserved. \href{https://www.ga4gh.org/cram/}{CRAM is the genomics compression standard for GA4GH}\\
\textbf{Compression:} NA\\
\textbf{Permissions:}\\
\textbf{Community access:}\\
\textbf{Location:}
\textbf{Notes:} When used with a reference genome, this exact reference genome file should be recorded.\\
\href{https://samtools.github.io/hts-specs/CRAMv3.pdf}{CRAM specification}

\begin{center}\rule{0.5\linewidth}{0.5pt}\end{center}

\hypertarget{processed-data}{%
\section{Processed data}\label{processed-data}}

\hypertarget{cgmap}{%
\subsection{CGmap}\label{cgmap}}

\textbf{File type:} CGmap\\
\textbf{Description:} TSV file of stranded pileup base calls for cytosine positions in the reference genome DNA methylation data.
\textbf{Compression:} gzip
\textbf{Permissions:}\\
\textbf{Access frequency:}\\
\textbf{Community access:}\\
\textbf{Identifiable:}\\
\textbf{Location:}\\
\textbf{Notes:} This file type is output from \href{https://github.com/BSSeeker/BSseeker2}{BSSeeker2} and used by \href{https://cgmaptools.github.io/}{CGmap tools}\\
\textbf{Sample:}

\begin{verbatim}
chr1    G   3000851 CHH CC  0.1 1   10
chr1    C   3001624 CHG CA  0.0 0   9
chr1    C   3001631 CG  CG  1.0 5   5
\end{verbatim}

Format descriptions (columns):

\begin{verbatim}
(1) chromosome \
(2) nucleotide on Watson (+) strand \
(3) position \
(4) context (CG/CHG/CHH) \
(5) dinucleotide-context (CA/CC/CG/CT) \
(6) methylation-level = #_of_C / (#_of_C + #_of_T) \
(7) #_of_C (methylated C, the count of reads showing C here) \
(8) = #_of_C + #_of_T (all Cytosines, the count of reads showing C or T here)
\end{verbatim}

\hypertarget{atcgmap}{%
\subsection{ATCGmap}\label{atcgmap}}

\textbf{File type:} ATCGmap\\
\textbf{Description:} TSV file of stranded pileup base calls for DNA methylation data.\\
\textbf{Compression:} gzip\\
\textbf{Permissions:}\\
\textbf{Access frequency:}\\
\textbf{Community access:}\\
\textbf{Identifiable:} YES\\
\textbf{Location:}\\
\textbf{Notes:} This file type is output from \href{https://github.com/BSSeeker/BSseeker2}{BSSeeker2} and used by \href{https://cgmaptools.github.io/}{CGmap tools}\\
\textbf{Sample:}

\begin{verbatim}
chr1    T   3009410 --  --  0   10  0   0   0   0   0   0   0   0   na
chr1    C   3009411 CHH CC  0   10  0   0   0   0   0   0   0   0   0.0
chr1    C   3009412 CHG CC  0   10  0   0   0   0   0   0   0   0   0.0
chr1    C   3009413 CG  CG  0   10  50  0   0   0   0   0   0   0   0.83
\end{verbatim}

Format descriptions (columns):

\begin{verbatim}
(1) chromosome \
(2) nucleotide on Watson (+) strand \
(3) position \
(4) context (CG/CHG/CHH) \
(5) dinucleotide-context (CA/CC/CG/CT) \
(6) - (10) plus strand \
(6) # of reads from Watson strand mapped here, support A on Watson strand \
(7) # of reads from Watson strand mapped here, support T on Watson strand \
(8) # of reads from Watson strand mapped here, support C on Watson strand \
(9) # of reads from Watson strand mapped here, support G on Watson strand \
(10) # of reads from Watson strand mapped here, support N \
(11) - (15) minus strand \
(11) # of reads from Crick strand mapped here, support A on Watson strand and T on Crick strand \
(12) # of reads from Crick strand mapped here, support T on Watson strand and A on Crick strand \
(13) # of reads from Crick strand mapped here, support C on Watson strand and G on Crick strand \
(14) # of reads from Crick strand mapped here, support G on Watson strand and C on Crick strand \
(15) # of reads from Crick strand mapped here, support N \
(16) methylation_level = #C/(#C+#T) = C8/(C7+C8) for Watson strand, =C14/(C11+C14) for Crick strand \
"nan" means none reads support C/T at this position. \
\end{verbatim}

\hypertarget{bigwig}{%
\subsection{Bigwig}\label{bigwig}}

\textbf{File type:} Bigwig\\
\textbf{Description:}\\
\textbf{Compression:}\\
\textbf{Permissions:}\\
\textbf{Access frequency:}\\
\textbf{Community access:}\\
\textbf{Identifiable:}\\
\textbf{Location:}\\
\textbf{Notes:}

\hypertarget{how-to-contribute}{%
\chapter{How to contribute}\label{how-to-contribute}}

Lorem ipsum

  \bibliography{book.bib,packages.bib}

\end{document}
