<!DOCTYPE html>
<html lang="en">
<head>
<meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
<meta charset="utf-8">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<title>Chapter 2 Data descriptors &amp; definitions | Data Standards - TKI Indigenous Genomics</title>
<meta name="author" content="Sam Buckberry, Jimmy Breen">
<meta name="description" content="This page provides definitions and guidance on the information that should be recorded for primary data files. Data that is clearly, simply and consistently described (named and defined) is much...">
<meta name="generator" content="bookdown 0.24 with bs4_book()">
<meta property="og:title" content="Chapter 2 Data descriptors &amp; definitions | Data Standards - TKI Indigenous Genomics">
<meta property="og:type" content="book">
<meta property="og:description" content="This page provides definitions and guidance on the information that should be recorded for primary data files. Data that is clearly, simply and consistently described (named and defined) is much...">
<meta name="twitter:card" content="summary">
<meta name="twitter:title" content="Chapter 2 Data descriptors &amp; definitions | Data Standards - TKI Indigenous Genomics">
<meta name="twitter:description" content="This page provides definitions and guidance on the information that should be recorded for primary data files. Data that is clearly, simply and consistently described (named and defined) is much...">
<!-- JS --><script src="https://cdnjs.cloudflare.com/ajax/libs/clipboard.js/2.0.6/clipboard.min.js" integrity="sha256-inc5kl9MA1hkeYUt+EC3BhlIgyp/2jDIyBLS6k3UxPI=" crossorigin="anonymous"></script><script src="https://cdnjs.cloudflare.com/ajax/libs/fuse.js/6.4.6/fuse.js" integrity="sha512-zv6Ywkjyktsohkbp9bb45V6tEMoWhzFzXis+LrMehmJZZSys19Yxf1dopHx7WzIKxr5tK2dVcYmaCk2uqdjF4A==" crossorigin="anonymous"></script><script src="https://kit.fontawesome.com/6ecbd6c532.js" crossorigin="anonymous"></script><script src="libs/header-attrs-2.11/header-attrs.js"></script><script src="libs/jquery-3.6.0/jquery-3.6.0.min.js"></script><meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<link href="libs/bootstrap-4.6.0/bootstrap.min.css" rel="stylesheet">
<script src="libs/bootstrap-4.6.0/bootstrap.bundle.min.js"></script><script src="libs/bs3compat-0.3.1/transition.js"></script><script src="libs/bs3compat-0.3.1/tabs.js"></script><script src="libs/bs3compat-0.3.1/bs3compat.js"></script><link href="libs/bs4_book-1.0.0/bs4_book.css" rel="stylesheet">
<script src="libs/bs4_book-1.0.0/bs4_book.js"></script><script src="https://cdnjs.cloudflare.com/ajax/libs/autocomplete.js/0.38.0/autocomplete.jquery.min.js" integrity="sha512-GU9ayf+66Xx2TmpxqJpliWbT5PiGYxpaG8rfnBEk1LL8l1KGkRShhngwdXK1UgqhAzWpZHSiYPc09/NwDQIGyg==" crossorigin="anonymous"></script><script src="https://cdnjs.cloudflare.com/ajax/libs/mark.js/8.11.1/mark.min.js" integrity="sha512-5CYOlHXGh6QpOFA/TeTylKLWfB3ftPsde7AnmhuitiTX4K5SqCLBeKro6sPS8ilsz1Q4NRx3v8Ko2IBiszzdww==" crossorigin="anonymous"></script><!-- CSS -->
</head>
<body data-spy="scroll" data-target="#toc">

<div class="container-fluid">
<div class="row">
  <header class="col-sm-12 col-lg-3 sidebar sidebar-book"><a class="sr-only sr-only-focusable" href="#content">Skip to main content</a>

    <div class="d-flex align-items-start justify-content-between">
      <h1>
        <a href="index.html" title="">Data Standards - TKI Indigenous Genomics</a>
      </h1>
      <button class="btn btn-outline-primary d-lg-none ml-2 mt-1" type="button" data-toggle="collapse" data-target="#main-nav" aria-expanded="true" aria-controls="main-nav"><i class="fas fa-bars"></i><span class="sr-only">Show table of contents</span></button>
    </div>

    <div id="main-nav" class="collapse-lg">
      <form role="search">
        <input id="search" class="form-control" type="search" placeholder="Search" aria-label="Search">
</form>

      <nav aria-label="Table of contents"><h2>Table of contents</h2>
        <ul class="book-toc list-unstyled">
<li><a class="" href="index.html"><span class="header-section-number">1</span> Introduction</a></li>
<li><a class="active" href="data-descriptors-definitions.html"><span class="header-section-number">2</span> Data descriptors &amp; definitions</a></li>
<li><a class="" href="metadata.html"><span class="header-section-number">3</span> Metadata</a></li>
<li><a class="" href="whole-genome-sequencing.html"><span class="header-section-number">4</span> Whole genome sequencing</a></li>
<li><a class="" href="dna-methylation.html"><span class="header-section-number">5</span> DNA methylation</a></li>
<li><a class="" href="reproducible-research.html"><span class="header-section-number">6</span> Reproducible research</a></li>
<li><a class="" href="how-to-contribute.html"><span class="header-section-number">7</span> How to contribute</a></li>
<li><a class="" href="references.html">References</a></li>
</ul>

        <div class="book-extra">
          
        </div>
      </nav>
</div>
  </header><main class="col-sm-12 col-md-9 col-lg-7" id="content"><div id="data-descriptors-definitions" class="section level1" number="2">
<h1>
<span class="header-section-number">2</span> Data descriptors &amp; definitions<a class="anchor" aria-label="anchor" href="#data-descriptors-definitions"><i class="fas fa-link"></i></a>
</h1>
<p>This page provides definitions and guidance on the information that should be recorded for primary data files. Data that is clearly, simply and consistently described (named and defined) is much easier for users to identify, understand, use and share the data. This also ensures consistency across projects, inter-operability and is complementary to reproducible research practices.</p>
<div id="describing-primary-data-file-types" class="section level2" number="2.1">
<h2>
<span class="header-section-number">2.1</span> Describing primary data file types<a class="anchor" aria-label="anchor" href="#describing-primary-data-file-types"><i class="fas fa-link"></i></a>
</h2>
<p>Biological data continue to be produced from an extensive array of assays and technologies. These data are often represented in unique file types and processed by specific software and algorithms. <strong>Below are descriptors that should be used for each file type.</strong></p>
</div>
<div id="file-type-descriptors" class="section level2" number="2.2">
<h2>
<span class="header-section-number">2.2</span> File type descriptors<a class="anchor" aria-label="anchor" href="#file-type-descriptors"><i class="fas fa-link"></i></a>
</h2>
<p><strong>File type:</strong> This is the file format to be typically denoted by the file extension.</p>
<p><strong>Description:</strong> A brief description of the file format.</p>
<p><strong>Compression:</strong> The algorithm and/or method used to compress the file.<br></p>
<p><strong>Permissions:</strong> The groups and users that have access to the data.<br></p>
<p><strong>Access frequency:</strong> How often this data type may require access. These descriptions are based on a subset of the <a href="https://cloud.google.com/storage/docs/storage-classes#classes">Google Cloud storage class descriptors</a>.</p>
<ul>
<li>
<strong>Standard:</strong> Standard Storage is for data that is frequently accessed and/or stored for only brief periods of time. Standard data files should be able to be regenerated from Nearline or Archive data files without too much time or effort.</li>
<li>
<strong>Nearline:</strong> Nearline Storage is ideal for data you plan to read or modify on average once per month or less. Nearline Storage is appropriate for data backup and short-term archiving.</li>
<li>
<strong>Archive:</strong> highly durable storage for data archiving, backup, and disaster recovery. Only used for infrequently accessed data.</li>
</ul>
<p><strong>Community access:</strong> Are their community custodians for these data? If so, who are they?<br></p>
<p><strong>Identifiable:</strong> Could this data be used to identify an individual (YES/NO).<br></p>
<p><strong>Location:</strong> Location(s) where the data should be stored.<br></p>
<p><strong>Notes:</strong> Any additional information that would assist users of the data.<br></p>
<p><strong>Sample:</strong> A sample of the file data with descriptions of each field. Useful if this file type is not a widely used and documented standard.</p>
<p>For example, the description the CGmap file format for DNA methylation data would look:</p>
</div>
<div id="example" class="section level2" number="2.3">
<h2>
<span class="header-section-number">2.3</span> Example<a class="anchor" aria-label="anchor" href="#example"><i class="fas fa-link"></i></a>
</h2>
<blockquote>
<p><strong>File type:</strong> CGmap<br><strong>Description:</strong> TSV file of stranded pileup base calls for cytosine positions in the reference genome DNA methylation data.<br><strong>Compression:</strong> gzip<br><strong>Permissions:</strong><br><strong>Access frequency:</strong><br><strong>Community access:</strong><br><strong>Identifiable:</strong><br><strong>Location:</strong><br><strong>Notes:</strong> This file type is output from <a href="https://github.com/BSSeeker/BSseeker2">BSSeeker2</a> and used by <a href="https://cgmaptools.github.io/">CGmap tools</a><br><strong>Sample:</strong></p>
<p>chr1 G 3000851 CHH CC 0.1 1 10<br>
chr1 C 3001624 CHG CA 0.0 0 9<br>
chr1 C 3001631 CG CG 1.0 5 5</p>
<p>Format descriptions (columns):<br>
(1) chromosome<br>
(2) nucleotide on Watson (+) strand<br>
(3) position<br>
(4) context (CG/CHG/CHH)<br>
(5) dinucleotide-context (CA/CC/CG/CT)<br>
(6) methylation-level = #_of_C / (#_of_C + #_of_T)<br>
(7) #_of_C (methylated C, the count of reads showing C here)<br>
(8) = #_of_C + #_of_T (all Cytosines, the count of reads showing C or T here)</p>
</blockquote>

</div>
</div>
  <div class="chapter-nav">
<div class="prev"><a href="index.html"><span class="header-section-number">1</span> Introduction</a></div>
<div class="next"><a href="metadata.html"><span class="header-section-number">3</span> Metadata</a></div>
</div></main><div class="col-md-3 col-lg-2 d-none d-md-block sidebar sidebar-chapter">
    <nav id="toc" data-toggle="toc" aria-label="On this page"><h2>On this page</h2>
      <ul class="nav navbar-nav">
<li><a class="nav-link" href="#data-descriptors-definitions"><span class="header-section-number">2</span> Data descriptors &amp; definitions</a></li>
<li><a class="nav-link" href="#describing-primary-data-file-types"><span class="header-section-number">2.1</span> Describing primary data file types</a></li>
<li><a class="nav-link" href="#file-type-descriptors"><span class="header-section-number">2.2</span> File type descriptors</a></li>
<li><a class="nav-link" href="#example"><span class="header-section-number">2.3</span> Example</a></li>
</ul>

      <div class="book-extra">
        <ul class="list-unstyled">
          
        </ul>
</div>
    </nav>
</div>

</div>
</div> <!-- .container -->

<footer class="bg-primary text-light mt-5"><div class="container"><div class="row">

  <div class="col-12 col-md-6 mt-3">
    <p>"<strong>Data Standards - TKI Indigenous Genomics</strong>" was written by Sam Buckberry, Jimmy Breen. It was last built on 2022-03-03.</p>
  </div>

  <div class="col-12 col-md-6 mt-3">
    <p>This book was built by the <a class="text-light" href="https://bookdown.org">bookdown</a> R package.</p>
  </div>

</div></div>
</footer>
</body>
</html>
